### DISCUSSION TITLE
Adding Elestio as deployment option

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [postalserver/postal#3133] [Feature suggestions]
Hey team,
I am Kaiwalya, Developer Advocate at Elestio. Elestio has been providing options of fully deploying and managing Postal application as shown [here](https://elest.io/open-source/postal). I think it would be a great idea if we can add it to official readme/documentation here.
🔔 In addition to this, if you are interested we provide partnership opportunities with tools we support by **revenue share** upon addition of this method in docs. If you would like to collaborate, just shoot me an email at [kaiwalya@elest.io](mailto:kaiwalya@elest.io) :)

**Comment by willpower232** [comment]
Postal has mostly shied away from recommending or listing server providers as they're all so different and get annoyed about sending emails in different ways. Also they're pretty attached to katapult.io by association so not sure what they would say but always great to see professional Postal support in the wider world.
